tests/monotouch-test/Security/CertificateTest.cs

```diff
@@ -585,5 +585,43 @@ public void MailX2 ()
 				}
 			}
 		}
+
+		[Test]
+		public void GenerateKeyPairTest ()
+		{
+			NSError error;
+			SecKey private_key;
+			SecKey public_key;
+			var att = new SecPublicPrivateKeyAttrs ();
+			att.Label = "NotDefault";
+			att.IsPermanent = true;
+			att.ApplicationTag = new NSData ();
+			att.EffectiveKeySize = 1024;
+			att.CanEncrypt = false;
+			att.CanDecrypt = false;
+			att.CanDerive = false;
+			att.CanSign = false;
+			att.CanVerify = false;
+			att.CanUnwrap = false;
+
+#if MONOMAC
```

Comment:
let's test the "single strongdict" API on XI as well
Just in case there's more difference than what's documented by Apple

Reply:
Yup I thought about it and added https://github.com/xamarin/xamarin-macios/pull/3438/files#diff-06456025d578358329b67590747b15a5R613 :P

See the commit itself for a clearer diff of my changes: https://github.com/xamarin/xamarin-macios/pull/3438/commits/c5deaaa34fce3442556451c56ce4a7e368310b61

Reply:
any reason not to have the test shared (instead of duplicated) for the same API ?
I would expect a single `#if IOS` to cover the iOS-specific overload

Reply:
Well I didn't about **that** (:

But yes, it would be similar to the source files.

Reply:
@spouliot fixed